dapps/src/tests/fetch.rs

```diff
@@ -167,13 +167,15 @@ fn should_return_fetched_dapp_content() {
 	response1.assert_status("HTTP/1.1 200 OK");
 	assert_security_headers_for_embed(&response1.headers);
 	assert_eq!(
```

Comment:
could use `assert!(condition, format, params..)` instead of `assert_eq!(expression, true, format, params..)`

Reply:
:+1: